src/thorlabs_cube/driver/kcube/ksc.py

```diff
@@ -0,0 +1,293 @@
+import struct as st
+
+from thorlabs_cube.driver.message import MGMSG, Message
+from thorlabs_cube.driver.tcube.tsc import Tsc, TscSim
+
+
+class Ksc(Tsc):
+    """
+    KSC101 K-Cube Solenoid Controller class
+    """
+
+    async def set_kcubemmi_params(
+        self,
+        js_mode: int,
+        js_max_vel: int,
+        js_accn: int,
+        dir_sense: int,
+        preset_pos1: int,
+        preset_pos2: int,
+        preset_pos3: int,
+        disp_brightness: int,
+        disp_timeout: int,
+        disp_dim_level: int,
+        js_sensitivity: int,
+    ) -> None:
+        """Set the KCube MMI (joystick) parameters.
+
+        :param js_mode: Joystick mode (1 = velocity control, 2 = jog mode, 3 = go-to-position mode).
+        :param js_max_vel: Maximum velocity for the joystick.
+        :param js_accn: Acceleration for the joystick.
+        :param dir_sense: Direction sense (normal or reversed).
+        :param preset_pos1: Preset position 1.
+        :param preset_pos2: Preset position 2.
+        :param preset_pos3: Preset position 3.
+        :param disp_brightness: Display brightness.
+        :param disp_timeout: Display timeout in ms.
+        :param disp_dim_level: Display dim level.
+        :param js_sensitivity: Joystick sensitivity.
+        """
+        payload = st.pack(
+            "<HHLLLHHLLLLH",
+            self._CHANNEL,
```

Comment:
```suggestion
            Ksc._CHANNEL,
```
I guess this is equivalent code, but I think this makes it more clear that the value is a class constant rather than an instance constant.